src/bst.py

```diff
@@ -32,6 +32,14 @@ def __init__(self, iterable=()):
             for item in iterable:
                 self.insert(item)
 
+    def balance(self, node):
+        """Return the balance of the bst."""
+        return (abs(max(self.left_depth, self.right_depth) - min(self.left_depth, self.right_depth)))
```

Comment:
By default your tree has no `left_depth` or `right_depth`. You wanted to check the depth on the input node